### DISCUSSION TITLE
Basic questions about zigbee

### DISCUSSION THREAD

**Discussion by mausch** [Koenkk/zigbee2mqtt#13692] [General]
Hi, last year my network died ( https://github.com/Koenkk/zigbee2mqtt/discussions/10228 ) and I haven't been able to get it back up since.

Since nothing I tried has made any difference at all (I get no logs at all anywhere, on any machine) I thought maybe I need to get a better understanding of how zigbee in general works first.

In particular around devices (all my devices are OSRAM / Ledvance lights if that makes any difference).
- If I reset a device, does it completely delete its state? e.g. channel, pan_id, extended_pan_id, network_key?
- When a device is pairing does it just broadcast on every existing channel?
- If I reset a device, and it doesn't pair, will it try to pair every time I turn it on again, until it pairs?
- If I reset a device, for how long will it try to join while it's on?
- If I delete all files in /var/lib/zigbee2mqtt, do I also need to reset anything in the adapter in order to completely reset the "server" side of things?

Many thanks

**Comment by DubhAd** [comment]
- A reset device is cleared of all setting, everything
- Unsure
- It will depend on the device in my experience. Mains devices usually yes, battery devices no.
- Varies by the device in my experience, but a few tens of seconds is a reasonable expectation
- Unsure
